integration-tests/src/test/java/tachyon/underfs/swift/SwiftUnderStorageCluster.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.underfs.swift;
+
+import java.io.IOException;
+import java.util.UUID;
+
+import tachyon.Constants;
+import tachyon.conf.TachyonConf;
+import tachyon.underfs.UnderFileSystem;
+import tachyon.underfs.UnderFileSystemCluster;
+
+/**
+ * This class will use SoftLayer or OpenStacl Swift as the backing store.
+ * The integration properties should be
+ * specified in the module's pom file. Each instance of the cluster will run with a separate base
+ * directory (user prefix + uuid). Each test will attempt to clean up their test directories, but
+ * in cases of complete failure (ie. jvm crashed) the directory will need to be cleaned up through
+ * manual means.
+ */
+public class SwiftUnderStorageCluster extends UnderFileSystemCluster {
+
+  private static final String INTEGRATION_SWIFT_API_KEY = "apiKey";
+  private static final String INTEGRATION_SWIFT_TENANT_KEY = "tenantKey";
+  private static final String INTEGRATION_SWIFT_USER_KEY = "userKey";
+  private static final String INTEGRATION_SWIFT_CONTAINER_KEY = "containerKey";
+  private static final String INTEGRATION_SWIFT_AUTH_METHOD_KEY = "authMethodKey";
+  private static final String INTEGRATION_SWIFT_AUTH_URL_KEY = "authUrlKey";
+
+  private String mSwiftContainer;
+
+  public SwiftUnderStorageCluster(String baseDir, TachyonConf tachyonConf) {
+    super(baseDir, tachyonConf);
+    String swiftAPIKey = System.getProperty(INTEGRATION_SWIFT_API_KEY);
```

Comment:
In general, we try to avoid using `System.getProperty` inside tachyon codebase---e.g., it is very hard for unittest. 

The good news is all system properties will be already loaded into `tachyonConf` passed to this constructor. So in here, you can simply do 
```java
String swiftAPIKey = tachyonConf.get(INTEGRATION_SWIFT_API_KEY);
```